x-pack/plugins/ml/public/embeddables/lens/lens_vis_layer_selection_flyout/style.scss

```diff
@@ -0,0 +1,23 @@
+// rough duplicates of lens styles
+// so our layer list looks similar
+
+.mlLensToJobFlyoutBody {
+  background-color: lightOrDarkTheme($euiColorLightestShade, $euiColorInk);
+}
+
+.mlLnsLayerPanel__layerHeader {
```

Comment:
I believe many of these can be removed if we switch to using the `EuiPanel` / `EuiSplitPanel`

Reply:
Changed in https://github.com/elastic/kibana/pull/129762/commits/82accec37a8e4b8790722f38f87fed1fae3deb55